test-network-function/networking/suite.go

```diff
@@ -275,3 +308,110 @@ func testNodePort(env *config.TestEnvironment) {
 		}
 	})
 }
+
+func getDeclaredPortList(command string, containerNum int, podName, podNamespace string) []PortDeclared {
```

Comment:
I don't think the command should be an argument. This function works with only one command. It's not generic.

Reply:
Same goes with the other function

Reply:
I agree, fixed.